client/app/src/main/java/com/example/criengine/Activities/BookActivity.java

```diff
@@ -0,0 +1,48 @@
+package com.example.criengine.Activities;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+import android.widget.EditText;
+import android.widget.ImageView;
+
+import com.example.criengine.R;
+
+public abstract class BookActivity extends AppCompatActivity {
+    EditText bookTitle;
+    EditText bookDetail;
+    EditText bookAuthor;
+    EditText bookISBN;
+    EditText bookStatus;
+    ImageView bookImage;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        // Must setContentView(layoutID) in child activity before calling super method
+        super.onCreate(savedInstanceState);
+
+        bookTitle = findViewById(R.id.bookView_title);
+        bookDetail = findViewById(R.id.bookView_detail);
+        bookAuthor = findViewById(R.id.bookView_author);
+        bookStatus = findViewById(R.id.bookView_status);
+        bookISBN = findViewById(R.id.bookView_ISBN);
+        bookImage = findViewById(R.id.bookView_image);
+
+        bookTitle.setTag(bookTitle.getKeyListener());
+        bookDetail.setTag(bookDetail.getKeyListener());
+        bookAuthor.setTag(bookAuthor.getKeyListener());
+    }
+
+
+    /**
+     * TODO:
+     *  - modes:
+     *      - request book (if book.status = available)
+     *      - add book to wishlist (if book.status = borrowed)
+     *      - manage book (if book.owner = account.username)
+     *          - buttons:
+     *              - "edit book"
+     *              - "see requests"
+     *              - "delete book" => confirmation popup?
+     */
```

Comment:
Will remove comments addressing PR comments

Reply:
A confirmation popup would be pretty good to have since if you want to undo the changes it would be a lot of work.
Do you think you can squeeze that popup into this PR or save it for later?

Reply:
already have that: https://github.com/CMPUT301F20T05/CriEngine/pull/56/files#diff-1c9884c60695d18efa967fe733c1b5adf213e15246a5a63b73ab7c850b000befR164

Reply:
I am the blind

But you could remove it from the TODO list you have (?)

Reply:
Yep, as stated above I will be removing the todo comment as I address PR comments